Backend/WebAPI/Controllers/ResetController.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Web.Http;
+using ServiceLayer.Services;
+using DataAccessLayer.Models;
+using ManagerLayer.ResetPassword;
+using ManagerLayer.UserManagement;
+
+namespace WebAPI.Controllers
+{
+    public class ResetController : ApiController
+    {
+
+        //After the user fills in the field with email, this action gets called
+        [HttpPost]
+        [ActionName("Send")]
+        public void SendResetEmail(string email)
```

Comment:
can you add the route header thing on top of each method so each route can be specified. 
[Route("api/reset")]